src/devices/Directory.Build.props

```diff
@@ -16,8 +17,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="UnitsNet" Version="$(UnitsNetPackageVersion)" />
-    <PackageReference Include="SixLabors.ImageSharp" Version="$(SixLaborsImageSharpPackageVersion)" />
     <ProjectReference Include="$(SystemDeviceModelPath)" Condition="'$(MSBuildProjectName)' != '$(SystemDeviceModelProjectName)'" />
+    <ProjectReference Include="$(MainLibraryPath)/System.Device.Gpio.csproj" />
```

Comment:
Why do we now include this project Reference here? Shouldn't this be includded by each device binding?